_includes/comparison.html

```diff
@@ -50,24 +50,58 @@ <h2 class="section-heading--primary"> Technical Comparison </h2>
         <td><a href="https://www.envoyproxy.io" target="_blank">Envoy</a></td>
       </tr>
       <tr>
-        <th scope="row">Platform</th>
-        <td>Kubernetes, Consul & Nomad</td>
-        <td>Kubernetes</td>
-        <td>Kubernetes, ECS, Fargate, EKS, EC2</td>
-        <td>Kubernetes, Nomad, standalone</td>
-        <td>Kubernetes</td>
-        <td>Kubernetes, “universal”</td>
-      </tr>
-      <tr>
-        <th scope="row">Automatic Sidecar Injection</th>
-        <td>yes</td>
-        <td>yes</td>
-        <td>yes</td>
-        <td>yes</td>
-        <td>yes (per Node)</td>
-        <td>yes</td>
-      </tr>
+          <th scope="row">Governance</th>
+          <td>see <a href="https://github.com/istio/community" target="_blank">Istio Community</a></td>
+          <td>see <a href="https://github.com/linkerd/linkerd2/blob/master/GOVERNANCE.md" target="_blank">Linkerd Governance</a> and <a href="https://github.com/cncf/foundation/blob/master/charter.md" target="_blank" >CNCF Charter</a></td>
+          <td>AWS</td>
+          <td>see <a href="https://github.com/hashicorp/consul/blob/master/.github/CONTRIBUTING.md" target="_blank">Contributing to Consul</a></td>
+          <td>see <a href="https://github.com/containous/maesh/blob/master/CONTRIBUTING.md" target="_blank">Contributing notice</a></td>
+          <td>see <a href="https://github.com/Kong/kuma/blob/master/CONTRIBUTING.md" target="_blank">Contributing notice</a></td>
+        </tr>
+      <tr>
+          <th scope="row">Tutorial</th>
+          <td><a href="https://istio.io/docs/tasks/" target="_blank">Istio Tasks</a></td>
+          <td><a href="https://linkerd.io/2/tasks/books/" target="_blank">Linkerd Tasks</a></td>
+          <td><a href="docs.aws.amazon.com/app-mesh/latest/userguide/appmesh-getting-started.html" target="_blank">AWS App Mesh Getting Started</a></td>
+          <td><a href="https://learn.hashicorp.com/consul/getting-started/connect" target="_blank">Hashicorp Learn platform</a></td>
+          <td><a href="https://docs.mae.sh/examples/" target="_blank">Maesh Example</a></td>
+          <td><a href="https://kuma.io/docs/0.2.2/installation/kubernetes/#kubernetes" target="_blank">Kuma Kubernetes Quickstart</a></td>
+        </tr>
     </tbody>
+    <thead style="background-color: white; color: #ff4d67">
+        <tr>
+          <th scope="col">Compatibility, Performance & Conplexity</th>
```

Comment:
s/Conplexity/Complexity
```suggestion
          <th scope="col">Compatibility, Performance & Complexity</th>
```

Reply:
Thank you! 👍